lib/linguist/languages.yml

```
@@ -6821,6 +6823,18 @@ Simple File Verification:
   codemirror_mode: properties
   codemirror_mime_type: text/x-properties
   language_id: 735623761
+Simula:
+  type: programming
+  color: "#B22F2F"
+  extensions:
+  - ".sim"
+  aliases:
+  - sim
+  tm_scope: source.simula
```

This should be the same scope name as the grammar definition:

```suggestion
  tm_scope: source.sim
```